samples/dynamodb/lambda/DynamoDB_1/Messages/OrderShipped.cs

```diff
@@ -0,0 +1,6 @@
+﻿using NServiceBus;
+
+public class OrderShipped : IEvent
+{
+  public string? OrderId { get; set; }
```

Comment:
thinking a bit more about this, should these properties (on all messages) really be marked nullable? From a consumer standpoint this doesn't make much sense. Does this cause issues when creating the messages?